### PR TITLE
fixed changes suggested by grammarly

### LINKED ISSUE #639: Run our *.md documents in this repo through the Grammer.ly service and update
#### Issue Summary

We would like to get our English polished up throughout the repo.

#### Aceptance Criteria

* Every .md file in this repo has been run through the Grammer.ly service and updated accordingly

### PR DESCRIPTION
**Pull request by xeon-zolt**
# Fixes #639 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:

- made changes in files as suggested by Grammarly app
```
├── CHANGELOG.md
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
│   ├── README.md
│   └── USAGE.md
│   └── README.md
│   └── mail-helper-refactor.md
├── README.md
│   │   │   ├── README.md
│   │   │   ├── README.md
│   │       ├── README.md
├── TROUBLESHOOTING.md
├── USAGE.md
    ├── asynchronous_mail_send.md
    ├── attachment.md
    ├── aws.md
    ├── django.md
    ├── domain_whitelabel.md
    ├── email_stats.md
    ├── error_handling.md
    ├── flask_heroku.md
    ├── README.md
    ├── slack_event_api_integration.md
    └── transational_templates.md
```

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=640) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=640) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=640) it.</sub>

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
- 
+
  ### Be Respectful
- Members of the community are respectful. We're respectful of others, their positions, their skills, their commitments and their efforts. We're respectful of the volunteer efforts that permeate the SendGrid community. We're respectful of the processes outlined in the community, and we work within them. When we disagree, we are courteous in raising our issues.  Overall, we're good to each other. We contribute to this community not because we have to, but because we want to. If we remember that, these guidelines will come naturally.
- 
+ Members of the community are respectful. We're respectful of others, their positions, their skills, their commitments ,and their efforts. We're respectful of the volunteer efforts that permeate the SendGrid community. We're respectful of the processes outlined in the community, and we work within them. When we disagree, we are courteous in raising our issues.  Overall, we're good with each other. We contribute to this community not because we have to, but because we want to. If we remember that, these guidelines will come naturally.
```
Comment (misterdorm) [inline CODE_OF_CONDUCT.md]: Nit: Misplaced space around comma at "commitments ,and their efforts."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -206,14 +206,14 @@ Please run your code through:
...
-3. Create a new topic branch (off the main project development branch) to
+3. Create a new topic branch (of the main project development branch) to
    contain your feature, change, or fix:
 
    ```bash
    git checkout -b <topic-branch-name>
    ```
 
-4. Commit your changes in logical chunks. Please adhere to these [git commit
+4. Commit your changes in logical chunks. Please adhere to these [git commits
```
Comment (misterdorm) [inline CONTRIBUTING.md]: 'commit' in this context is a noun, not a verb.  Trailing 's' not needed.

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ If you are using a [requirements file](https://pip.readthedocs.io/en/1.1/require
 <a name="versions"></a>
 ## Versioning Convention
 
-We follow the MAJOR.MINOR.PATCH versioning scheme as described by [SemVer.org](http://semver.org). Therefore, we recommend that you always pin (or vendor) the particular version you are working with to your code and never auto-update to the latest version. Especially when there is a MAJOR point release, since that is guaranteed to be a breaking change. Changes are documented in the [CHANGELOG](https://github.com/sendgrid/sendgrid-python/blob/master/CHANGELOG.md) and [releases](https://github.com/sendgrid/sendgrid-python/releases) section.
+We follow the MAJOR.MINOR.PATCH versioning scheme as described by [SemVer.org](http://semver.org). Therefore, we recommend that you always pin (or vendor) the particular version you are working with your code and never auto-update to the latest version. Especially when there is a MAJOR point release since that is guaranteed to be a breaking change. Changes are documented in the [CHANGELOG](https://github.com/sendgrid/sendgrid-python/blob/master/CHANGELOG.md) and [releases](https://github.com/sendgrid/sendgrid-python/releases) section.
```
Comment (misterdorm) [inline TROUBLESHOOTING.md]: Change to "...working with in your code...", feels more natural than the original "to your code".  But only removing 'to' is not appropriate.

File: USAGE.md [hunk 4]
```diff
@@ -70,7 +70,4628 @@ IP Access Management allows you to control which IP addresses can be used to acc
 
 For more information, please see our [User Guide](http://sendgrid.com/docs/User_Guide/Settings/ip_access_management.html).
 
-### POST /access_settings/whitelist
```
Comment (misterdorm) [inline USAGE.md]: Was this change a mistake?

File: use_cases/aws.md [hunk 5]
```diff
@@ -2,14 +2,14 @@
 
 This tutorial explains how to set up a simple "Hello Email" app on AWS, using the AWS CodeStar service.
 
-We'll be creating a basic web service to send email via SendGrid. The application will run on AWS Lambda, and the "endpoint" will be via AWS API Gateway.
+We'll be creating a basic web service to send an email via SendGrid. The application will run on AWS Lambda, and the "endpoint" will be via AWS API Gateway.
```
Comment (misterdorm) [inline use_cases/aws.md]: 'email' is plural in this context, 'an' not needed.

File: use_cases/aws.md [hunk 6]
```diff
@@ -2,14 +2,14 @@
...
 
-The neat thing is that CodeStar provides all of this in a pre-configured package. We just have to make some config changes, and push our code.
+The neat thing is that CodeStar provides all of this in a pre-configured package. We just have to make some config changes and push our code.
 
-Once this tutorial is complete, you'll have a basic web service for sending email that can be invoked via a link to your newly created API endpoint.
+Once this tutorial is complete, you'll have a basic web service for sending an email that can be invoked via a link to your newly created API endpoint.
```
Comment (misterdorm) [inline use_cases/aws.md]: 'email' is plural in this context, 'an' not needed.

File: use_cases/aws.md [hunk 7]
```diff
@@ -19,18 +19,18 @@ Before starting this tutorial, you will need to have access to an AWS account in
...
+Log in to your AWS account and go to the AWS CodeStar service. Click "Start a project". For this tutorial we're going to choose a Python Web service, utilizing AWS Lambda. You can use the filters on the left-hand side of the UI to narrow down the available choices.
 
-After you've selected the template, you're asked to provide a name for your project. Go ahead and name it "hello-email". Once you've entered a name, click "Create Project" in the lower right hand corner. You can then choose which tools you want to use to interact with the project. For this tutorial, we'll be choosing "Command Line". 
+After you've selected the template, you're asked to provide a name for your project. Go ahead and name it "hello-email". Once you've entered a name, click "Create Project" in the lower right-hand corner. You can then choose which tools you want to use to interact with the project. For this tutorial, we'll be choosing the "Command Line".
```
Comment (misterdorm) [inline use_cases/aws.md]: 'the' not appropriate in this context.

File: use_cases/aws.md [hunk 8]
```diff
@@ -19,18 +19,18 @@ Before starting this tutorial, you will need to have access to an AWS account in
...
 
-Once that is completed, you'll be given some basic steps to get Git installed and setup, and instructions for connecting to the AWS CodeCommit(git) repository. You can either use HTTPS, or SSH. Instructions for setting up either are provided. 
+Once that is completed, you'll be given some basic steps to get Git installed and setup, and instructions for connecting to the AWS CodeCommit(git) repository. You can either use HTTPS or SSH. Instructions for setting up either are provided. 
 
-Go ahead and clone the Git repository link after it is created. You may need to click "Skip" in the lower right hand corner to proceed.
+Go ahead and clone the Git repository link after it is created. You may need to click "Skip" in the lower right-hand corner to proceed.
 
-Once that's done, you've successfully created a CodeStar project! You should be at the dashboard, with a view of the wiki, change log, build pipeline, and application endpoint. 
+Once that's done, you've successfully created a CodeStar project! You should be at the dashboard, with a view of the wiki, changelog, build a pipeline, and application endpoint.
```
Comment (misterdorm) [inline use_cases/aws.md]: 'build' is a noun in this context, 'a' not needed.

File: use_cases/aws.md [hunk 9]
```diff
@@ -169,4 +169,4 @@ SENDGRID_API_KEY
 
 Now, go back to your project dashboard in CodeStar. Click on the link under "Application endpoints". After a moment, you should be greeted with JSON output indicating an email was successfully sent. 
 
-Congratulations, you've just used serverless technology to create an email sending app in AWS!
\ No newline at end of file
+Congratulations, you've just used serverless technology to create an email sending the app in AWS!
```
Comment (misterdorm) [inline use_cases/aws.md]: 'sending' is noun here, 'the' not needed.

File: use_cases/django.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# Create a Django app to send email with SendGrid
+# Create a Django app to send an email with SendGrid
```
Comment (misterdorm) [inline use_cases/django.md]: 'email' is plural, 'an' not needed.

File: use_cases/django.md [hunk 11]
```diff
@@ -119,7 +119,7 @@ $ python manage.py migrate
 $ python manage.py runserver
 ```
 
-By default, it starts the development server at `http://127.0.0.1:8000/`. To test if we can send email or not, go to `http://127.0.0.1:8000/sendgrid/`. If it works, we should see the page says "Email Sent!".
+By default, it starts the development server at `http://127.0.0.1:8000/`. To test if we can send an email or not, go to `http://127.0.0.1:8000/sendgrid/`. If it works, we should see the page says "Email Sent!".
```
Comment (misterdorm) [inline use_cases/django.md]: 'email' is plural, 'an' not needed.

File: use_cases/flask_heroku.md [hunk 12]
```diff
@@ -1,4 +1,4 @@
-# Create a Flask app to send email with SendGrid
+# Create a Flask app to send an email with SendGrid
```
Comment (misterdorm) [inline use_cases/flask_heroku.md]: 'email' is plural, 'an' not needed.

**Comment by misterdorm** [comment]
Thanks @xeon-zolt !   Could you please address/explain the large amount of changes to USAGE.md?

**Comment by xeon-zolt** [comment]
@misterdorm please check again

**Comment by thinkingserious** [comment]
Hello @xeon-zolt,

Could you please resolve the merge conflicts? Thank you!

With Best Regards,

Elmer

**Comment by thinkingserious** [comment]
Hello @xeon-zolt,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @xeon-zolt,<br /><br/>Thanks again for the PR!<br /><br />                                         We noticed that you have earned 3 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold.<br /><br />                                         Also, please be sure that you have officially registered <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @xeon-zolt,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 3 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request merged**